syntax = "proto2";

package rpc;

message RpcMeta {
    required string service_name = 1;
    required string method_name = 2;
    required int32 data_size = 3;
}